framework/src/mysql.js

```diff
@@ -116,7 +117,8 @@ const mapRowsBySchema = async (rawRows, schema) => {
 		const row = {};
 		Object.keys(schema).forEach(o => {
 			const val = item[o];
-			if (val || val === 0 || val === false) row[o] = getValue(cast(val, schema[o].type));
+			const valType = schema[o].type;
+			if (`${o}` in item) row[o] = getValue(cast(val, valType));
```

Comment:
maybe we can cast inside the `getValue` method itself. so the call will be more neat here. ex: `getValue(val, valType)`. Also, renaming `o` to `key` would be nice.